Gemfile

```diff
@@ -16,6 +16,7 @@ gem 'i18n-js', '3.5.1' # A library to provide the I18n translations on the Javas
 gem 'fast_jsonapi' # A lightning fast JSON:API serializer for Ruby Objects
 gem 'rack-cors' # Provides support for CORS
 gem 'graphql' # Ruby implementation of GraphQL
+gem 'gravtastic' # Gravatars for Rails
```

Comment:
This gem is not in active development, last commit was 6 years ago and the repository is now archived.

It can be done easily without gem
```Ruby
def gravatar_url(email)
    gravatar_id = Digest::MD5::hexdigest(email)
    "https://secure.gravatar.com/avatar/#{gravatar_id}"
end
```
https://blog.kiprosh.com/using-gravatar-profile-image-in-ruby-on-rails-application/

Reply:
@junan Thanks for suggestion 💯